composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": "^7.0",
         "doctrine/annotations": "^1.0",
-        "phpdocumentor/reflection-docblock": "^3.0|^4.0",
+        "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0",
```

Comment:
`*`?

Reply:
I am not a fan of `*`. 

Restrictions are good. =)

Reply:
Consistency also ;)

Reply:
Joke appart, I don't like that packs add their own constraints, that's why I prefer a `*`. Ppl that need better control should unpack.

Reply:
Yeah. That is fair. 

I’m okey with * but only because this is a pack. Other packages (that require the documentor) need to have “real” versions specified in their dependencies.

Reply:
Totally!

Reply:
The difference is that other `*` constraints won't actually run as `*` in a Flex project, as they are filtered by the Symfony Flex constraint. This one is really a `*`.